src/check.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_CHECK_H
+#define BITCOIN_CHECK_H
+
+#include <cstdio>
+#include <cstdlib>
+
+// CHECK(expression)
+// =================
+// * If compiled with -DABORT_ON_FAILED_CHECK (set by --enable-debug):
+//   Evaluate expression and abort if expression is falsy.
+//
+// * If compiled without -DABORT_ON_FAILED_CHECK:
```

Comment:
Shorter without loss of precision:

```suggestion
// * otherwise:
```

Reply:
Good point! Fixed!